### ISSUE TITLE
Editor: Deal with long lines better

### ISSUE THREAD

**Issue by TerryCavanagh** [TerryCavanagh/autosurvive#68]
Right now they basically don't work at all

**Comment by randomnine** [comment]
How about we limit lines to the width of the panel? It's probably clearer if we don't support long lines at all

**Comment by TerryCavanagh** [comment]
Yeah, that's definitely the easiest solution! Hmm, will come back to this...
